src/dvclive/live.py

```diff
@@ -548,6 +548,10 @@ def end(self):
             catch_and_warn(DvcException, logger)(ensure_dir_is_tracked)(
                 self.dir, self._dvc_repo
             )
+            if self._dvcyaml:
+                catch_and_warn(DvcException, logger)(self._dvc_repo.scm.add)(
```

Comment:
This is missing some test

Reply:
I meant the `scm.add`, it seems that we only test the `include_untracked` change

Reply:
Good catch. Added it to `test_untracked_dvclive_files_inside_dvc_exp_run_are_added`.